CMakeLists.txt

```diff
@@ -80,10 +80,7 @@ target_compile_definitions(pluginval PRIVATE
     JUCE_PLUGINHOST_LV2=1
     JUCE_USE_CURL=0
     JUCE_WEB_BROWSER=0
-    JUCER_ENABLE_GPL_MODE=1
```

Comment:
This flag would only affect the Projucer, and then only in older JUCE versions. There's no need to set it on other JUCE projects